tenant-cd-api/src/main/java/org/junit/jupiter/api/extension/support/package-info.java

```diff
@@ -0,0 +1,9 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+/**
+ * @author jonmv
+ */
+@ExportPackage(version = @Version(major = 5, minor = 8, micro = 1))
+package org.junit.jupiter.api.extension.support;
```

Comment:
This one seems to have just been forgotten when the original re-exports were added.